crates/bevy_ui/src/ui_node.rs

```diff
@@ -2800,11 +2800,16 @@ impl<'w, 's> DefaultUiCamera<'w, 's> {
 }
 
 /// Derived information about the camera target for this UI node.
+///
+/// Updated in [`UiSystems::Prepare`](crate::UiSystems::Prepare) by [`update_ui_context_system`](crate::update::update_ui_context_system)
 #[derive(Component, Clone, Copy, Debug, Reflect, PartialEq)]
 #[reflect(Component, Default, PartialEq, Clone)]
 pub struct ComputedNodeTarget {
+    /// The id of the target camera for this UI node.
     pub(crate) camera: Entity,
+    /// The scale factor of the target camera's render target.
     pub(crate) scale_factor: f32,
+    /// The size of the target camera's viewport in physical pixels.
```

Comment:
Does this ignore the scale factor, like for `RenderTargetInfo::physical_size`?

Reply:
Yes that's right.